sw/device/silicon_creator/manuf/skus/earlgrey_a0/generic/BUILD

```diff
@@ -88,74 +89,59 @@ opentitan_ram_binary(
         "//sw/device/silicon_creator/manuf/lib:flash_info_fields",
         "//sw/device/silicon_creator/manuf/lib:individualize",
         "//sw/device/silicon_creator/manuf/lib:otp_fields",
-        "//sw/device/silicon_creator/manuf/lib:sram_program_linker_script",
         "//sw/device/silicon_creator/manuf/lib:sram_start",
     ],
 )
 
-opentitan_functest(
+opentitan_test(
     name = "cp_provision",
-    srcs = ["//sw/device/silicon_creator/rom/e2e:empty_test.c"],
-    cw310 = cw310_params(
+    cw310 = cw310_jtag_params(
+        binaries = {
+            ":sram_cp_provision": "sram_cp_provision",
+        },
         bitstream = "//hw/bitstream:rom_with_fake_keys_otp_raw",
         tags = ["manuf"],
-        test_cmds = [
-            "--clear-bitstream",
-            "--bitstream=\"$(location {bitstream})\"",
-            "--elf=\"$(rootpath :sram_cp_provision_fpga_cw310_elf_transition)\"",
-            "--all-steps",
-        ] + OPENTITANTOOL_OPENOCD_TEST_CMDS,
+        test_cmd = """
+            --elf={sram_cp_provision}
+            --all-steps
```

Comment:
what happened to the "--clear-bitstream" cmd? is this not required anymore? It was originally added because this test modifies OTP.

Reply:
same below

Reply:
@cfrantz moved it to `OPENTITANTOOL_OPENOCD_TEST_CMD` (together with `--bitstream=...`) which is included in `cw310_jtag_params`. I have mixed feelings about this, on the one hand it avoid a lot of repetition, but it's also not very clear.

Reply:
ah I see. really we only want the clear-bitstream command to be applied to tests that alter OTP images, since these could persist between tests causing errors, but for performance we would like to not have to reload a bitstream every time if not required. I suppose we can revisit this later.